js/src/builders.ts

```diff
@@ -25,8 +25,8 @@ export function createMessage<T extends keyof MessageTypes, U extends keyof Mess
     id,
     threadId,
     parentId,
-    to          : opts.to ?? opts.last.to,
-    from        : opts.from ?? opts.last.from,
+    to          : opts.from ?? opts.last.from,
```

Comment:
This reads pretty confusingly to someone without context, maybe add a comment explaining why they need to be swapped?